ui/README.md

```diff
@@ -1,30 +1,23 @@
-# React + TypeScript + Vite
+# Web UI
```

Comment:
Does it make sense to squash this change into the commit that introduced this file?